charts/akeyless-k8s-secrets-injection/templates/webhook-servicaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.serviceAccount.create -}}
+{{- if or (.Values.serviceAccount.create) (.Values.restartRollout.enabled) }}
```

Comment:
what if the user entered his own SA (and used the "name" field)? will the new role binding match the SA?